examples/text_classification/example_custom_dataset.py

```diff
@@ -0,0 +1,7 @@
+from autogluon import TextClassification as task
+
+dataset = task.Dataset(filepath='/home/ubuntu/data/tsv/train.csv', usecols=['text', 'target'])
```

Comment:
what if the username is not ubuntu? where is the train.csv file?

Reply:
Agreed, all example files should be runnable, additionally they should have relative paths. Expect the user's working directory to be the autogluon root directory.

Reply:
Also this `train.csv` should probably actually live in the autogluon s3 bucket, like our example datasets for TabularPrediction:

https://github.com/awslabs/autogluon/blob/master/examples/tabular/example_simple_tabular.py